src/main/kotlin/no/nav/syfo/personstatus/infrastructure/database/repository/PersonOversiktStatusRepository.kt

```diff
@@ -268,9 +274,28 @@ class PersonOversiktStatusRepository(private val database: DatabaseInterface) :
         }
     }
 
+    // Burde vi bare ha en put operasjon for person_oversikt_status tabellen?
+    override fun updatePersonstatusesWithNavnAndFodselsdato(personer: List<PersonOversiktStatus>): List<PersonOversiktStatus> =
```

Comment:
Her burde vi kanskje ha en felles `PUT` aktig funksjon? 🤔 (Som "bytter" ut databaseraden med nye verdier selv om man i praksis i noen tilfeller bare bytter ut én eller to rader.) Tanker?

Reply:
Hm, godt poeng. Nedsiden er at det kan bli veldig mange spesifikke funksjoner vs. én oppdateringsfunksjon. Vet ikke hva som er god praksis her. Burde jo finnes noe `synchronized` aktig funksjon i kotlin som det gjør i java 🤔
Dette var uansett bare en tanke til senere, så lar det være nå 😊

Reply:
Ja, det å ha spesifikkse update-funksjoner her reduserer risikoen for at vi får samtidighetskonflikter som gjør at gamle data overskriver nye.